gnomad/utils/vep.py

```diff
@@ -303,44 +304,44 @@ def process_consequences(
     :param csq_order: Optional list indicating the order of VEP consequences, sorted
         from high to low impact. Default is None, which uses the value of the
         `CSQ_ORDER` global.
+    :param has_polyphen: Whether the input VEP Struct has PolyPhen and SIFT.
+        Default is True. Note: VEP105-annotated context HT still has PolyPhen inside
+        vep struct but removed from the VEP105-annotated release HTs, so this has to be
+        set to False if using gnomAD v4.0 and later release HTs.
     :return: MT with better formatted consequences.
     """
     if csq_order is None:
         csq_order = CSQ_ORDER
     csqs = hl.literal(csq_order)
     csq_dict = hl.literal(dict(zip(csq_order, range(len(csq_order)))))
 
+    def csq_score(tc: hl.expr.StructExpression) -> int:
```

Comment:
I think we should make these "private" with a _ prefix since they only make sense inside this function. We haven't always done this through the repo, this function included but I think its a helpful thing to add.